cineast-runtime/src/main/java/org/vitrivr/cineast/standalone/Sandbox.java

```diff
@@ -0,0 +1,125 @@
+                "20160914_145746_000.jpg              ",
+                "20160914_145803_000.jpg              ",
+                "20160914_145820_000.jpg              ",
+                "20160914_145837_000.jpg              ",
+                "20160914_145854_000.jpg              ",
+                "20160914_145911_000.jpg              ",
+                "20160914_145928_000.jpg              ",
+                "20160914_145958_000.jpg              ",
+                "B00011200_21I6X0_20180507_212447E.JPG",
+                "B00011201_21I6X0_20180507_212559E.JPG",
+                "B00011202_21I6X0_20180507_212707E.JPG",
+                "B00011203_21I6X0_20180507_212819E.JPG",
+                "B00011204_21I6X0_20180507_212928E.JPG",
+                "B00011205_21I6X0_20180507_213036E.JPG",
+                "B00011206_21I6X0_20180507_213147E.JPG",
+                "B00011207_21I6X0_20180507_213256E.JPG",
+                "B00011208_21I6X0_20180507_213407E.JPG",
+                "B00011209_21I6X0_20180507_213736E.JPG"
+        };
+        String[] expected = new String[]{
+                "20160914_1457",
+                "20160914_1458",
+                "20160914_1458",
+                "20160914_1458",
+                "20160914_1458",
+                "20160914_1459",
+                "20160914_1459",
+                "20160914_1459",
+                "20180507_2124",
+                "20180507_2125",
+                "20180507_2127",
+                "20180507_2128",
+                "20180507_2129",
+                "20180507_2130",
+                "20180507_2131",
+                "20180507_2132",
+                "20180507_2134",
+                "20180507_2137"
+        };
+
+        for(int i=0; i<items.length; i++){
+            String minuteId = LSCUtilities.filenameToMinuteId(items[i]).get();
+            System.out.println("Given: "+items[i]+", Result: "+minuteId+" = "+minuteId.equalsIgnoreCase(expected[i])+" ("+expected[i]+")");
+            System.out.println("  Time: "+LSCUtilities.fromMinuteId(minuteId));
+        }
+        */
+        // Testing the path remover
+        /*
+        String[] items = new String[]{
+                "lsc2020-images/2018-05-07/B00011202_21I6X0_20180507_212707E.JPG",
+                "lsc2020-images/2018-05-07/B00011203_21I6X0_20180507_212819E.JPG",
+                "lsc2020-images/2018-05-07/B00011204_21I6X0_20180507_212928E.JPG",
+                "lsc2020-images/2018-05-07/B00011205_21I6X0_20180507_213036E.JPG",
+                "lsc2020-images/2018-05-07/B00011206_21I6X0_20180507_213147E.JPG",
+                "lsc2020-images/2018-05-07/B00011207_21I6X0_20180507_213256E.JPG",
+                "lsc2020-images/2018-05-07/B00011208_21I6X0_20180507_213407E.JPG",
+                "lsc2020-images/2018-05-07/B00011209_21I6X0_20180507_213736E.JPG"
+        };
+        for(String s : items){
+            System.out.println(s+" => "+sanitizeFilename(s));
+        }*/
+    }
+}

```

Comment:
This should be deleted before the merge.

Reply:
Stuff that slipped through my cleanup step - sure, this will be removed.